src/backoffice/core/components/property-type-based-property/property-type-based-property.element.ts

```diff
@@ -93,7 +93,7 @@ export class UmbPropertyTypeBasedPropertyElement extends UmbLitElement {
 			this._dataTypeObserver = this.observe(
 				await this._dataTypeRepository.byId(dataTypeId),
 				(dataType) => {
-					this._dataTypeData = dataType?.values || [];
+					this._dataTypeData = new UmbDataTypePropertyCollection(dataType?.values || []);
```

Comment:
Maybe there is a detail I don't see here, but wouldn't UmbDataTypePropertyCollection fallback to a empty array if the given value is 'undefined'.

aka. i don't think you need the ` || [] `here

Reply:
I'll double check, but I do remember something exploding if I didn't include the empty array fallback.

Reply:
All works fine without the fallback 👍